park_linac.py

```diff
@@ -179,7 +186,9 @@ def delta_func():
                 raise CavityHWModeError(f"{self} not Online, Maintenance, or Ready")
 
             self.check_resonance()
-            self.steppertuner.move_to_cold_landing(count_current=count_current)
+            self.steppertuner.move_to_cold_landing(
+                count_current=count_current, check_detune=False
+            )
```

Comment:
I think I get that you are using false/true when you want to check for detune? 
Maybe add a comment or two in the if sections on why you check or don't for certain situations?

Reply:
Do you think this explanation is clear?